presto-docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -28,8 +28,7 @@ Java Runtime Environment
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
 * 64-bit required
-* version 11 recommended (lower versions not supported, higher versions less
-  tested)
+* version 11.0.7+ (version 12+ not supported)
```

Comment:
This removes the explicit call out that lower versions are not supported. We specifically added this in the past since we got multiple questions to clarify ..

Reply:
Followed up in https://github.com/prestosql/presto/pull/5958

Reply:
I think "not supported" in this context is confusing. The "not supported" for versions below 11 is not the same as the "not supported" for versions above 11. I think it's the subtle difference between "The Presto software doesn't support it (i.e., it won't work)" vs "We don't provide support for it (it may or may not work, but you're on your own".

Reply:
The current master has this clarified in a text